django/db/models/functions/text.py

```diff
@@ -70,6 +70,16 @@ class Lower(Transform):
     lookup_name = 'lower'
 
 
+class Ltrim(Transform):
```

Comment:
Can they be named LTrim, RTrim, LStrip, RStrip? I don't think we ever have lowercase words ("trim", "strip") in class names. Admittedly they are classes that are used more like functions, but the same is true for all the existing ones.

Reply:
Maybe `Ntile`, but I can't figure out what it stands for...

Reply:
Agreed.